README.md

```diff
@@ -254,20 +254,11 @@ const WebSocket = require('ws');
 
 const wss = new WebSocket.Server({ port: 8080 });
 
-// Broadcast to all.
-wss.broadcast = function broadcast(data) {
-  wss.clients.forEach(function each(client) {
-    if (client.readyState === WebSocket.OPEN) {
-      client.send(data);
-    }
-  });
-};
-
 wss.on('connection', function connection(ws) {
   ws.on('message', function incoming(data) {
     // Broadcast to everyone else.
     wss.clients.forEach(function each(client) {
-      if (client !== ws && client.readyState === WebSocket.OPEN) {
+      if (client.readyState === WebSocket.OPEN) {
```

Comment:
This is not ok as it now broadcasts to ALL clients including the sender of the message.

Reply:
Oh I see.
I couldn't figure it out from the code having looked at it for the first time.
In this context `client` is actually a client's web socket object and that wasn't obvious to me.

Got no obvious suggestion to make it more readable I'm afraid. :confused: